internal/daemon/controller/controller.go

```diff
@@ -414,13 +416,38 @@ func (c *Controller) AuthorizeNodeeWorker(request string) error {
 	const op = "controller.(Controller).AuthorizeWorker"
```

Comment:
Could this function have documentation?

Reply:
This function is a temporary function until the create PR lands. I'll actually probably merge Jim's PR first and then remove it.